### DISCUSSION TITLE
2023-retrospect/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#177] [Blog comments]
# 2023-retrospect/

  

https://hyeon9mak.github.io/2023-retrospect/

**Comment by bperhaps** [comment]
놀때는 진짜 병X같이 놀지만 알고보면 너무 멋진사람.
한해 수고했다~~~

**Reply by Hyeon9mak** [reply]
후버댐도 5년만에 완공했는데 우리도 곧...!
너잘도 2023년 고생많았쓰요 ㅋㅋㅋ
